app/routes/events/utils.ts

```diff
@@ -62,9 +62,9 @@ export const EventTypeConfig: Record<EventType, ConfigProperties> = {
     textColor: '#FFF',
   },
   [EventType.KiD_EVENT]: {
-    displayName: 'KiD-arrangement',
-    color: 'var(--color-event-black)',
-    textColor: 'var(--color-white)',
+    displayName: 'NEXUS-arrangement',
+    color: '#00509E',
+    textColor: '#FFF',
```

Comment:
Could we still use our white text color? If you always want it to be white (even on dark mode) you can use `var(--color-absolute-white)`